### DISCUSSION TITLE
meshopt support in Scene Viewer

### DISCUSSION THREAD

**Discussion by vogloblinsky** [google/model-viewer#4022] [Q&A]
Hi,

While reading this page, i cannot clearly see if Scene Viewer on Android support or will support meshopt compression ?

https://developers.google.com/ar/develop/scene-viewer#file_requirements_for_models

The model i have compressed works fine in the editor of modelviewer because the editor loads meshopt decoder (https://github.com/google/model-viewer/pull/3089); but not in an Android device in Scene Viewer.

**Comment by elalish** [comment]
I tend to think not. @AdrianAtGoogle may have more info.

**Comment by AdrianAtGoogle** [comment]
The documentation is out of date, but Scene Viewer supports meshopt as of fall 2022; if a meshopt model is not loading correctly on the latest version of the android google app please send a link to the file so we can address any issues.
